sql/wsrep_trans_observer.h

```diff
@@ -193,6 +193,11 @@ static inline bool wsrep_run_commit_hook(THD* thd, bool all)
                        wsrep_is_active(thd), wsrep_is_real(thd, all),
                        wsrep_has_changes(thd), wsrep_thd_is_applying(thd),
                        wsrep_is_ordered(thd)));
+
+  /* No need to run commit hooks if wsrep is not enabled */
+  if (!thd->variables.wsrep_on)
+    DBUG_RETURN(false);
+
   /* Is MST commit or autocommit? */
```

Comment:
MySQL version seems to have here:

  /* skipping non-wsrep threads */
  if (!WSREP(thd)) return (false);

Better to use same code in both sides

Reply:
Fixed